backend/models/userProfile.model.js

```diff
@@ -0,0 +1,109 @@
+const mongoose = require("mongoose");
+
+mongoose.Promise = global.Promise;
+
+const userProfileSchema = mongoose.Schema({
+  firstName: { type: String },
+  lastName: { type: String },
```

Comment:
I am thinking that this is required as well, but I would love to hear people's opinion on that.

Reply:
I think @alex-anakin or someone with requirements info should look into.

Reply:
Unfortunately, we can't make the fields required because we need to save parts of the user profile data step-by-step during the onboarding process.